### PR TITLE
SEO samples for WC PieChart and DoughnutChart

### PR DESCRIPTION
**Pull request by igDivya**
SEO samples for WC PieChart and DoughnutChart

### CODE REVIEW

File: samples/charts/doughnut-chart/explosion/src/DoughnutChartExplosion.ts [hunk 1]
```diff
@@ -8,70 +10,42 @@ import { ModuleManager } from 'igniteui-webcomponents-core';
...
-    private chart: IgcDoughnutChartComponent;
-    private label: HTMLElement;
+    private chart: IgcDoughnutChartComponent; 
 
     constructor() {
 
-        let ringSeries = document.getElementById('ringSeries') as IgcRingSeriesComponent;
-        ringSeries.explodedSlices.add(3);
-        ringSeries.explodedSlices.add(4);
+        let ringSeries = document.getElementById('ringSeries') as IgcRingSeriesComponent;
```
Comment (HUSSAR-mtrela) [inline samples/charts/doughnut-chart/explosion/src/DoughnutChartExplosion.ts:23]: why did you remove this code in DoughnutChart Explosion sample?
ringSeries.explodedSlices.add(3);	
ringSeries.explodedSlices.add(4);

File: samples/charts/pie-chart/explosion/src/PieChartExplosion.ts [hunk 2]
```diff
@@ -16,6 +18,8 @@ export class PieChartExplosion {
         this.chart = document.getElementById('chart') as IgcPieChartComponent;
         this.chart.dataSource = this.getData();
         this.chart.sliceClick = this.onSliceClick;
+        this.chart.legend = document.getElementById('legend') as IgcItemLegendComponent;
+
```
Comment (HUSSAR-mtrela) [inline samples/charts/pie-chart/explosion/src/PieChartExplosion.ts:22]: this sample is missing code for exploding slices

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: samples/charts/doughnut-chart/explosion/src/DoughnutChartExplosion.ts (hunk 1 above)
Comment (igDivya) [reply samples/charts/doughnut-chart/explosion/src/DoughnutChartExplosion.ts:23]: added back.

File: samples/charts/pie-chart/explosion/src/PieChartExplosion.ts (hunk 2 above)
Comment (igDivya) [reply samples/charts/pie-chart/explosion/src/PieChartExplosion.ts:22]: slice is exploded through html : exploded-slices="1, 2"

File: samples/charts/doughnut-chart/explosion/src/DoughnutChartExplosion.ts (hunk 1 above)
Comment (HUSSAR-mtrela) [reply samples/charts/doughnut-chart/explosion/src/DoughnutChartExplosion.ts:23]: it is still missing

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

**Review by HUSSAR-mtrela — APPROVED** [review]
